crates/re_space_view_spatial/src/parts/assets3d.rs

```diff
@@ -45,12 +45,7 @@ impl Asset3DPart {
             .and_then(|mut batch| batch.next());
 
         let mesh = Asset3D {
-            data: arch_view
-                .iter_required_component::<Blob>()?
-                .next()
-                .ok_or_else(|| re_types::DeserializationError::MissingData {
-                    backtrace: re_types::_Backtrace::new_unresolved(),
-                })?,
+            data: arch_view.required_mono_component::<Blob>()?,
```

Comment:
Seems weird to me that having _too much_ information will now fail to render anything: this should be a warning.